src/components/App.js

```diff
@@ -1,9 +1,14 @@
 import React from 'react';
+import { Provider } from 'react-redux';
+import store from '../store';
+
 
 const App = () => (
-  <div>
-    <h1>Authors Haven</h1>
-  </div>
+  <Provider store={store}>
```

Comment:
Since this is a component, can you place the Provider component inside `src/index.js` instead? It's best to keep all configuration in one place.

Reply:
I have worked on this feedback